obspy/taup/__init__.py

```diff
@@ -185,6 +184,63 @@
     TauPyModel().get_ray_paths(
         500, 130, phase_list=["ttbasic"]).plot(plot_type="cartesian")
 
+Travel times for these ray paths can be plotted using the
+:meth:`~obspy.taup.tau.Arrivals.timeplot` method:
+
+>>> arrivals = model.get_ray_paths(source_depth_in_km=500,
+...                                distance_in_degree=130)
+>>> arrivals.timeplot()  # doctest: +SKIP
+
+.. plot::
+    :width: 50%
+    :align: center
+
+    from obspy.taup import TauPyModel
+    TauPyModel().get_ray_paths(500, 130).timeplot()
```

Comment:
Could you use only a couple of phases here? Otherwise the plot's legend just get really big and the plot not as pretty as it could be.